src/inspector_agent.cc

```diff
@@ -205,8 +202,33 @@ std::string GenerateID() {
   return uuid;
 }
 
-}  // namespace
+std::string StringViewToUtf8(const StringView& view) {
+  if (view.is8Bit())
+    return std::string(reinterpret_cast<const char*>(view.characters8()),
+        view.length());
+
+  const uint16_t* source = view.characters16();
+  std::string result(view.length() * sizeof(*source), '\0');
+  UnicodeString utf16(reinterpret_cast<const UChar*>(source), view.length());
```

Comment:
Is this reinterpret_cast necessary?  If for some reason UChar != uint16_t, then this line should arguably fail to compile instead of silently doing the wrong thing.